src/libs/actions/Policy/Policy.ts

```diff
@@ -1038,6 +1038,9 @@ function updateGeneralSettings(policyID: string, name: string, currencyValue?: s
     const customUnitID = distanceUnit?.customUnitID;
     const currency = currencyValue ?? policy?.outputCurrency ?? CONST.CURRENCY.USD;
 
+    const currencyPendingAction = currency !== policy?.outputCurrency ? CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE : null;
+    const namePendingAction = name !== policy?.name ? CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE : null;
```

Comment:
@dominictb 

1. Go offline
2. Change name
3. Change the currency
--> Both the name and the currency should be greyed out 

In this case, the pendingFields.name is overwriten with `null` value. We can use undefined to avoid overwriting
https://github.com/user-attachments/assets/47c1c301-e6ea-4cb9-af50-e950126684f5